docs/src/components/scalar.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react";
+import { ApiReference as VueComponent } from "@scalar/api-reference";
+import { useTheme } from "nextra-theme-docs";
+import { applyVueInReact } from "veaury";
+
+const ApiReference = applyVueInReact(VueComponent);
+
+const specUrl =
+  "https://uploadthing-git-swagger-pinglabs.vercel.app/openapi.json";
```

Comment:
Should point to prod url

Reply:
yea, need this merged first: https://github.com/pingdotgg/uploadthing-infra/pull/283